### PR TITLE
DOC-184 and updating variable reference

### PR DESCRIPTION
**Pull request by azhebel**
Ensuring alphabetical order, fixing typos, some formatting tweaks.

### CODE REVIEW

File: doc/source/wsrep-provider-index.rst [hunk 1]
```diff
@@ -193,7 +193,7 @@ This variable defines how often to retransmit EVS join messages when forming clu
    :dyn: No
    :default: PT1S 
 
-This variable defines how often will keepalive beacons will be emmited (in the absence of any other traffic).
+This variable defines how often to emmit keepalive beacons (in the absence of any other traffic).
```
Comment (kbauskar) [inline doc/source/wsrep-provider-index.rst]: s/emmit/emit/gc

File: doc/source/wsrep-provider-index.rst [hunk 2]
```diff
@@ -555,7 +557,7 @@ When this variable is set to ``TRUE`` node will completely ignore the quorum cal
    :dyn: Yes
    :default: false
 
-When this variable us set ti ``TRUE`` node will process updates even in the case of split brain. This should be used with extreme caution in multi-master setup, but should simplify things in master-slave cluster (especially if only 2 nodes are used).
+When this variable us set to ``TRUE``, the node will process updates even in the case of a split brain. This should be used with extreme caution in multi-master setup, but should simplify things in master-slave cluster (especially if only 2 nodes are used).
```
Comment (kbauskar) [inline doc/source/wsrep-provider-index.rst]: s/us set to/is set to/gc

File: doc/source/wsrep-provider-index.rst [hunk 3]
```diff
@@ -585,7 +587,7 @@ When this variable is set to ``TRUE`` more recent primary component overrides ol
    :dyn: No
    :default: true
 
-When this variable is set to ``true`` the node stores the Primary Component state to disk. The Primary Component can then recover automatically when all nodes that were part of the last saved state re-establish communications with each other. This feature allows automatic recovery from full cluster crashes, such as in the case of a data center power outage and graceful full cluster restarts without the need for explicitly bootstrapping a new Primary Component.
+When this variable is set to ``true``, the node stores the Primary Component state to disk. The Primary Component can then recover automatically when all nodes that were part of the last saved state re-establish communications with each other. This feature allows automatic recovery from full cluster crashes, such as in the case of a data center power outage. A subsequent graceful full cluster restart will the require explicit bootstrapping for a new Primary Component.
```
Comment (kbauskar) [inline doc/source/wsrep-provider-index.rst]: s/will the require explicit/will require explict/g

File: doc/source/wsrep-system-index.rst [hunk 4]
```diff
@@ -1,23 +1,8 @@
...
-
-**NOTE**: Prior to |Percona XtraDB Cluster| :rn:`5.6.24-25.11`, :variable:`wsrep_OSU_method` was only a global variable. Current behavior is now consistent with |MySQL| behavior for variables that have both global and session scope. This means if you want to change the variable in current session you need to do it with: ``SET wsrep_OSU_method`` (without the ``GLOBAL`` keyword). Setting the variable with ``SET GLOBAL wsrep_OSU_method`` will change the variable globally but it won't have effect on current session.
+===============================
+Index of wsrep system variables
+===============================
```
Comment (kbauskar) [inline doc/source/wsrep-system-index.rst:5]: Did we removed wsrep_OSU_method description or moved it to some other place ?

**Comment by kbauskar** [comment]
I see most of the changes are fixing grammatically errors. There is no content change as such.
Looks good to me with the said trivial review comments.

Fix the review comments and go ahead with merge. No need of second review.

File: doc/source/wsrep-system-index.rst (hunk 4 above)
Comment (azhebel) [inline doc/source/wsrep-system-index.rst:5]: moved alphabetically

**Pull request merged**